editor/src/clj/editor/editor_extensions/coerce.clj

```diff
@@ -102,7 +102,7 @@
                     (comp
                       (map-indexed
                         (fn [i coercer]
-                          (coercer vm (.rawget x (unchecked-inc-int i)))))
+                          (coercer vm (vm/with-lock vm (.rawget x (unchecked-inc-int i))))))
```

Comment:
Would it be more efficient to acquire the lock once, evaluate all the coercers, then release the lock?

Reply:
Not sure what's more efficient, but I think semantically it will be better to use a single lock per table iteration, so other threads can't change the table. I fixed it now (+ I made `one-of` more performant in a happy-path)